share/spice/rx_info/rx_info.js

```diff
@@ -28,16 +28,14 @@
         var script = $('[src*="/js/spice/rx_info/"]')[0],
             source = $(script).attr("src"),
             triggerWordMatch = parseInt(source.match(/rx_info\/[^\/]+\/(\d)/)[1]),
-            relCheck;
-        
-
-        // meta information
-        var sourceName = "More at DailyMed",
-            sourceUrl  = "http://dailymed.nlm.nih.gov/",
+            relCheck,
+            sourceName = "DailyMed",
+            sourceUrl = "http://dailymed.nlm.nih.gov/",
```

Comment:
This link should ideally direct the user to more search results -- can we add a search parameter to this URL?